### PR TITLE
Fix first deploy cluster creation

### PR DESCRIPTION
**Pull request by MatheusdeMelo**
Fix the ECS cluster creation in the first deploy on CodeBuild.

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -25,6 +25,11 @@ while [ : ]; do
 			#echo "Repository slug is '$2'"
```
Comment (fagianijunior) [inline provision.sh:25]: unnecessary comment

File: provision.sh [hunk 2]
```diff
@@ -25,6 +25,11 @@ while [ : ]; do
...
 			shift 2
 			;;
+        -i | --cluster-id)
+            provision_cluster_id=$2
+            #echo "Cluster ID is '$2'"
```
Comment (fagianijunior) [inline provision.sh:30]: unnecessary commnet

**Review by fagianijunior — CHANGES_REQUESTED** [review]
Needs to remove the unnecessary comment on code.

**Comment by MatheusdeMelo** [comment]
> Needs to remove the unnecessary comment on code.

With the commit "Remove comments" this requested has be changed.

**Review by fagianijunior — APPROVED** [review]
]ok

**Pull request merged**